pkg/api/v1alpha1/cluster_webhook.go

```diff
@@ -343,3 +346,40 @@ func (r *Cluster) ValidateDelete() error {
 
 	return nil
 }
+
+func validateServerVersionSkew(new, old *Cluster) field.ErrorList {
+	var allErrs field.ErrorList
+
+	if old.IsSelfManaged() {
+		return allErrs
+	}
+
+	path := field.NewPath("spec")
```

Comment:
shouldn't this be `field.NewPath("spec", "kubernetesVersion")`?

Reply:
The Kubernetes version is added for those individual fields when returning error at the end of the method.